stubs/zoloto/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Zoloto."""
+
+from enum import IntEnum
+
+__version__: str
+
+
+class MarkerDict(IntEnum):
```

Comment:
Why is this necessary? The provided types should be more than enough

Reply:
Due to the way that mypy stubs work, it will completely ignore any types in your library unfortunately. The stubs can be removed at a later date when upstream has the hints and implements PEP 561